src/index.ts

```diff
@@ -255,7 +352,7 @@ export class ActionsAPI {
    * @returns The parcel detail, including ids for dictionaries it contains
    */
   async readParcel(id: number): Promise<ReadParcelResult> {
```

Comment:
per discussion with @joshhaug and @mattdailis we decided to change this to not take an argument & instead always return the *single parcel* associated with your workspace